### ISSUE TITLE
Not working with homebridge 1.30

### ISSUE THREAD

**Issue by hoefs** [ToddGreenfield/homebridge-nut#24]
Does not want to start. When disabling, homebridge works fine

**Comment by HGordonH** [comment]
Also experiencing the same. Had to disable this plugin to start 1.3.0

**Comment by rangerek** [comment]
yup +1 from my side as well - moved to separate HB to avoid issues with HB start - waiting for a fix

**Comment by HGordonH** [comment]
Judging by other comments in the open issues this plugin appears to be no longer supported so if anyone know alternatives it would be great to know.

**Comment by ffaamm** [comment]
Has someone tried [vectronic/homebridge-nut](https://github.com/vectronic/homebridge-nut)?

**Comment by ffaamm** [comment]
And it seems to work fine - or at least it installs and configure fine, I still haven’t tested a blackout.

**Comment by HGordonH** [comment]
I ended up switching to Homebridge Apcaccess. Works fine on an APC UPS.

**Comment by Ewarren7** [comment]
> Has someone tried [vectronic/homebridge-nut](https://github.com/vectronic/homebridge-nut)?

Just did, thanks for pointing out this new fork. My exact config I was using now works on this fork with homebridge 1.34
